parcel/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "main": "../lib/cjs/parcel.cjs",
```

Comment:
The module format for this main field should match the format for the single-spa-react package.json (UMD)

Reply:
Done.